<launch>
  <env name="ROSCONSOLE_FORMAT" value="[${severity}][${thread}][${node}/${function}:${line}]: ${message}"/>
  <env name="ROS_LOG_DIR" value="/tmp/"/>
  
  <node pkg="robot_simplified" type="robot_simplified" name="robot_simplified" output="screen" respawn="true">
  
    <param name="serial_dev" value="/dev/serial/by-id/usb-Teensyduino_USB_Serial_9936370-if00" />
    <param name="baudrate" type="int" value="115200"/>

    <param name="tf2_rate" type="int" value="100"/>
    <param name="tf2_input_topic" value="/move_base_simple/goal" />
    <param name="tf2_base_link" value="/base_link" />
    <param name="tf2_map_link" value="/map" />

  </node>

</launch>
